koku/providers/aws/provider.py

```diff
@@ -136,13 +136,17 @@ def cost_usage_source_is_reachable(self, credentials, data_source):
             internal_message = f"Unable to access account resources with ARN {credential_name}."
             raise serializers.ValidationError(error_obj(key, internal_message))
 
-        s3_exists = _check_s3_access(storage_resource_name, creds)
+        region_kwargs = {}
+        if region_name := data_source.get("bucket_region"):
+            region_kwargs["region_name"] = region_name
+
+        s3_exists = _check_s3_access(storage_resource_name, creds, **region_kwargs)
```

Comment:
If this is not easy to read let me know and I'll just set the default here.

The purpose is to avoid having a default value of `us-east-1` set here as well as function definitions. That's just a lot of places to have a default value floating around.

Reply:
I think it looks fine to me!